srcpkgs/blender/patches/disable_werror.patch

```diff
@@ -6,8 +6,8 @@ index f22688f..c32aad1 100644
  
  # Warnings
  if(CMAKE_COMPILER_IS_GNUCXX)
--	ADD_CHECK_CXX_COMPILER_FLAG(CMAKE_CXX_FLAGS _has_cxxflag_float_conversion "-Werror=float-conversion")
--	ADD_CHECK_CXX_COMPILER_FLAG(CMAKE_CXX_FLAGS _has_cxxflag_double_promotion "-Werror=double-promotion")
- 	ADD_CHECK_CXX_COMPILER_FLAG(CMAKE_CXX_FLAGS _has_no_error_unused_macros "-Wno-error=unused-macros")
- 	unset(_has_cxxflag_float_conversion)
- 	unset(_has_cxxflag_double_promotion)
+-  ADD_CHECK_CXX_COMPILER_FLAG(CMAKE_CXX_FLAGS _has_cxxflag_float_conversion "-Werror=float-conversion")
+-  ADD_CHECK_CXX_COMPILER_FLAG(CMAKE_CXX_FLAGS _has_cxxflag_double_promotion "-Werror=double-promotion")
```

Comment:
Instead of making them non-fatal, these warnings can be fixed by updating `openimageio` ([mvf/blender-2.90.1](https://github.com/mvf/void-packages/commits/blender-2.90.1)) This requires a revbump of `luxmark` and `luxrays` though and they seem borked.

Reply:
I pulled your `openimageio` bump and see what you mean about `luxmark` and `luxrays`. For the moment I will defer addressing this issue because the upstream sources for `luxmark` and `luxrays` seem to have moved and it's not clear to me where the new repos are. (I see a `luxmark` repo on GitHub, but `luxrays` seems to have disappeared.)

Because `blender` is currently broken with system Python, I'm going to merge this as-is and let somebody else propose a solution for moving to a new `openimageio` in the future.